### DISCUSSION TITLE
Support Request: Fedora 41 build failed

### DISCUSSION THREAD

**Discussion by withoutme432** [OpenTabletDriver/OpenTabletDriver#3562] [General]
### What do you need help with?

hi again sorry for the previous duplicate support request
this time i got 

Creating package with type 'RedHat'...
Running package generation script: /home/dt/Downloads/OpenTabletDriver/eng/linux/RedHat/package.sh

RPMizing...
Making a source tarball...
Generating opentabletdriver.spec...
Moving /home/dt/Downloads/OpenTabletDriver/dist/redhat to /home/dt/Downloads/OpenTabletDriver/dist/redhat/opentabletdriver...
/home/dt/Downloads/OpenTabletDriver/eng/linux/RedHat/package.sh: line 96: rpmbuild: command not found
Build failed!
maybe it has someting to do with dnf version being updated to version 5 on fedora 41

### What have you tried?

building

### Diagnostic Information

.

### Acknowledgements

- [X] I have installed OpenTabletDriver by following its [official installation instructions](https://opentabletdriver.net/).
- [X] I have checked the [OpenTabletDriver Wiki](https://opentabletdriver.net/Wiki) and respective FAQ pages, and my issue was not covered or actually fixed.
- [X] I have searched the existing issues and this new issue is not a duplicate of any.
- [X] I have written a concise and meaningful title.
- [X] I am on the latest version of OpenTabletDriver.

**Comment by jamesbt365** [comment]
> rpmbuild: command not found

It is telling you what your system is missing, search up this error using a search engine and figure out what package that is installed with.

**Comment by withoutme432** [comment]
`/home/dt/Downloads/OpenTabletDriver/eng/linux/RedHat/package.sh: line 96: rpmbuild: command not found`
maybe it's about dnf5's commands changing

**Comment by withoutme432** [comment]
`rpmbuild -D "_topdir ${moved_output}" -bb "${moved_output}/SPECS/${OTD_LNAME}.spec"` that's the line in question

**Reply by jamesbt365** [reply]
Well you can't run the rpmbuild command without installing the package that contains said command.

**Reply by withoutme432** [reply]
what's the command in question

**Reply by jamesbt365** [reply]
`rpmbuild` is the command, if you don't understand what commands are on linux yet, you should probably read more documentation or wait until the next OpenTabletDriver release which targets dotnet 8.

**Reply by withoutme432** [reply]
got it to install thank you for your support
